core/jobs_test.py

```diff
@@ -95,7 +95,7 @@ def test_base_job_manager_enqueue_raises_error(self):
         with self.assertRaisesRegexp(
             NotImplementedError,
             'Subclasses of BaseJobManager should implement _real_enqueue().'):
-            jobs.BaseJobManager._real_enqueue(  # pylint: disable=protected-access
+            jobs.BaseJobManager.real_enqueue(  # pylint: disable=protected-access
```

Comment:
Remove the pylint disable comment (`# pylint: disable=protected-access`).

Reply:
I have removed pylint disabled comment for all the tests I have fixed

Reply:
I don't see it?

Reply:
you've removed it from the other file, but not here

Reply:
I have pushed the new code.